dts/bindings/sensor/ti,tmp1075.yaml

```diff
@@ -0,0 +1,55 @@
+# Copyright (c) 2024 Arrow Electronics.
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+  TMP1075 Digital Temperature Sensor.  See more info at
+  https://www.ti.com/product/TMP1075
+
+compatible: "ti,tmp1075"
+
+include: [sensor-device.yaml, i2c-device.yaml]
+
+properties:
+  conversion-rate:
+    description: Conversion rate in ms.
+    type: int
+    default: 220
```

Comment:
Need to explain in the description why the default value was selected
https://docs.zephyrproject.org/latest/build/dts/bindings-upstream.html#dt-bindings-default-rules

Reply:
removed default value and updated overlay file